packages/mobile/src/screens/product-info/components/header/header.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FC } from 'react';
+import { HeaderButton, View, Text } from '~/components/components';
+import { useCustomTheme } from '~/hooks/hooks';
+import { globalStyles } from '~/styles/styles';
+import { styles } from './styles';
+
+type ItemHeaderProps = {
+  onBackPress: () => void;
+};
+
+const Header: FC<ItemHeaderProps> = ({ onBackPress }) => {
```

Comment:
@AleksandrTermenzhy shouldn't we use native header there?

Reply:
fix